resources/channel.sub.js

```diff
@@ -31,7 +31,20 @@
     }
 
 
-    // TODO: should be a way to clean up unused sockets
+    /**
+     * Cache of WebSocket instances per channel
+     *
+     * For reading there can only be one channel with each UUID, so we
+     * just have a simple map of {uuid: WebSocket}. The socket can be
+     * closed when the channel is closed.
+     *
+     * For writing there can be many channels for each uuid. Those can
+     * share a websocket (within a specific global), so we have a map
+     * of {uuid: [WebSocket, count]}.  Count is incremented when a
```

Comment:
Great docs, thanks! In particular noting when the count is decremented is useful.